test/K6/src/tests/maskinporten/maskinporten.js

```diff
@@ -108,8 +108,6 @@ export function getMaskinPortenSchemaOfferedTest() {
     'get offered MaskinPortenSchemas - performedByUserId is ${performedByUserId}': (r) => r.json('0.performedByUserId') == org1.dagl.userid,
```

Comment:
Can also add verification of all properties of the response model. Currently missing:
offeredByName
offeredByOrganizationNumber

Reply:
added these checks